core/src/lib.rs

```diff
@@ -78,6 +78,9 @@ pub const STACK_TRACE_OFFSET: usize = DECODER_TRACE_RANGE.end;
 pub const STACK_TRACE_WIDTH: usize = 19;
 pub const STACK_TRACE_RANGE: Range<usize> = range(STACK_TRACE_OFFSET, STACK_TRACE_WIDTH);
 
+// Range check trace table
+pub const RANGE_CHECK_TRACE_TABLE_WIDTH: usize = 256;
+
```

Comment:
For consistency, I don't think this is the best place for this constant. The top level here contains stuff that is a constant describing the shape of the trace (main or auxiliary) or globally used. In general, we have created specific modules within this crate for each VM component that has constants which are used in multiple crates (e.g. `decoder`, `stack`, etc). However, when constants aren't needed in multiple crates, we tend to just define them in the module/crate where they're needed. 

In this case, this constant should probably be defined within the `range` module in the `processor` crate

Reply:
Done!